app/src/repositories/user/userRepository.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const pool = require("../../config/db");
+
+class UserRepository {
+  async findByUserId(user_id) {
+    const [rows] = await pool.query("SELECT * FROM users WHERE user_id = ?", [user_id]);
```

Comment:
mysql 드라이버에서 세미콜론 처리를 해줘서 문제가 없을거같긴한데 SQL 표준 문법은 쿼리마다  세미콜론을 붙여줘야하기에 명시적으로 처리해주는게 좋을거같습니다

```suggestion
    const [rows] = await pool.query("SELECT * FROM users WHERE user_id = ?;", [user_id]);
```

Reply:
아래 두 메서드도 동일한 코멘트입니다

Reply:
좋은 피드백 감사합니다!
말씀해주신 대로 세미콜론을 명시해주는 게 더 명확할 것 같습니다.
기존 코드에 순차적으로 세미콜론을 추가해서 일관되게 관리해보겠습니다!